plugins/experimental/brotli_transform/README

```diff
@@ -0,0 +1,7 @@
+Using Brotli Algorithm to do compression
+https://github.com/google/brotli
+
+If the origin server return gzip content, do gzip inflate first
+
+libbrotli
```

Comment:
Why do we have this dependency? it should be completely possible to do this with only the dependency on Google's code.